charts/jenkins/templates/jenkins-master-secondary-ingress.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.master.secondaryingress.enabled }}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
Is this still needed? This has been required for a long time

Reply:
its pretty common and doesn't seem to harm anything to have it.